pkg/kubeapiserver/options/plugins.go

```diff
@@ -137,6 +137,7 @@ func DefaultOffAdmissionPlugins() sets.String {
 		mutatingwebhook.PluginName,          //MutatingAdmissionWebhook
 		validatingwebhook.PluginName,        //ValidatingAdmissionWebhook
 		resourcequota.PluginName,            //ResourceQuota
+		podpriority.PluginName,              //PodPriority
```

Comment:
only add this if the PodPriority feature gate is on?

Reply:
You mean, import kube_features and check if podPriority is enabled and then append podpriority.PluginName to list?

Reply:
yes

Reply:
I am confused! Isn't this the `DefaultOff` list?!

Reply:
this is computing DefaultOff by taking all registered plugins and subtracting the defaultOn list

Reply:
Ok, nevermind. The function name confused me.

Reply:
> this is computing DefaultOff by taking all registered plugins and subtracting the defaultOn list

Yes, I actually figured before seeing your comment. It is a very odd combination that we build the defaultOn list in the defaultOff function!